internal/assets/views/open-source-licences.html

```
@@ -0,0 +1,1789 @@
+
+<section class="container mx-auto px-4 py-8">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)